zc/zdaemonrecipe/tests.py

```diff
@@ -66,7 +66,7 @@ def newlines_in_program():
         path /sample-buildout/parts/run/transcript.log
       </logfile>
     </eventlog>
-    """
+    """  # noqa: F821
```

Comment:
I'm starting to understand @jamadden's aversion to these ;)

F821 is "undefined name", and I think we could instead tell flake8 about the special names that zc.buildout's test setup makes available to test.globs, by adding a 

```
[flake8]
builtins = cat, system, join, write  # and whatever
```

to setup.cfg.

Reply:
see https://github.com/zopefoundation/zc.zdaemonrecipe/pull/5